docs(doc): add cloud section and rectify faq (#890)
instill-ai/instill.tech: docs/cloud/credit.en.mdx

| OpenAI | TTS 1    | 150                           |
| OpenAI | TTS 1 HD | 300                           |

#### Next Additions

Check our [Roadmap](../roadmap) and subscribe to our
[Newsletter](/newsletter/?utm_source=documentation&utm_medium=link) to keep up
to date with the latest tasks, models and vendors that are supported by
**Instill Credit**.

We also plan to leverage **Instill Credit** to ease data storage and model
hosting, stay tuned!

## Organization Credit

Team plans also include monthly **Instill Credit** for organization, allowing

> their users running a pipeline owned by an organization they belong to and they

Reviewer: ```suggestion
their users to run pipelines owned by an organization they belong to and they
```

> their users running a pipeline owned by an organization they belong to and they

Reviewer: ```suggestion
their users to run pipelines owned by an organization they belong to, without
```

> won't see their **Instill Credit** affected. Instead, the organization's credit

Reviewer: ```suggestion
their own **Instill Credit** being affected. Instead, the organization's credit
```
